.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: ruby
 cache: bundler
 rvm:
-  - 2.4
```

Comment:
@eldemcan Could you also change the minimum version specified in the gemspec? [here](https://github.com/mokhan/spandx/blob/ea60a6a21bdc3a9f96fa76c8f35e1334dde8de9c/spandx.gemspec#L17).